docs/AboutUs.md

```diff
@@ -3,8 +3,9 @@
 
 | Display                                             |           Name            |                Github Profile                |              Portfolio               |
 |-----------------------------------------------------|:-------------------------:|:--------------------------------------------:|:------------------------------------:|
-| ![](https://via.placeholder.com/100.png?text=Photo) |     Wilson Ng Jing An     |        [Github](https://github.com/wilsonngja)         |   [Portfolio](team/wilsonngja.md)    |
-| ![](https://via.placeholder.com/100.png?text=Photo) |      Syed Omar Zoran      |   [Github](https://github.com/zoranabc201)   |  [Portfolio](team/syedomarzoran.md)  |
+
+| ![](https://via.placeholder.com/100.png?text=Photo) |     Wilson Ng Jing An     | [Github](https://github.com/wilsonngja)      |   [Portfolio](team/wilsonngja.md)    |
+| ![](https://via.placeholder.com/100.png?text=Photo) |      Syed Omar Zoran      | [Github](https://github.com/zoranabc201)     |  [Portfolio](team/zoranabc201.md)    |
```

Comment:
![image](https://user-images.githubusercontent.com/31720838/200154431-9d33f51b-10ee-407e-88c6-2f09e1348eac.png)

It seems like when you are editing, there's some issue with the formatting.

Reply:
Fixed in new PR.